main.less

```diff
@@ -1,32 +1,25 @@
+/*Font family for errorPanel*/
+@fontFamily: Helvetica, Arial, "Meiryo UI", "ＭＳ Ｐゴシック", "MS PGothic", sans-serif;
+
 .errorHighlight {
     background-color: #EDC5C5;
 }
 
-.errorButton {
-    margin: 1px;
-    background-color: #FF0000;
-}
-
-.errorText {
```

Comment:
pretty sure this needs to be there. It styles the text in the box containing error details.